src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -71,4 +89,95 @@ public enum JsonTokenType : byte
+        [System.CLSCompliantAttribute(false)]
+        public void WriteNumber(System.ReadOnlySpan<char> propertyName, ulong value, bool suppressEscaping = false) { }
+        public void WriteNumber(string propertyName, decimal value, bool suppressEscaping = false) { }
+        public void WriteNumber(string propertyName, double value, bool suppressEscaping = false) { }
+        public void WriteNumber(string propertyName, int value, bool suppressEscaping = false) { }
+        public void WriteNumber(string propertyName, long value, bool suppressEscaping = false) { }
+        public void WriteNumber(string propertyName, float value, bool suppressEscaping = false) { }
+        [System.CLSCompliantAttribute(false)]
+        public void WriteNumber(string propertyName, uint value, bool suppressEscaping = false) { }
+        [System.CLSCompliantAttribute(false)]
+        public void WriteNumber(string propertyName, ulong value, bool suppressEscaping = false) { }
+        public void WriteNumberValue(decimal value) { }
+        public void WriteNumberValue(double value) { }
+        public void WriteNumberValue(int value) { }
+        public void WriteNumberValue(long value) { }
+        public void WriteNumberValue(float value) { }
+        [System.CLSCompliantAttribute(false)]
+        public void WriteNumberValue(uint value) { }
+        [System.CLSCompliantAttribute(false)]
+        public void WriteNumberValue(ulong value) { }
+        public void WriteStartArray() { }
+        public void WriteStartArray(System.ReadOnlySpan<byte> propertyName, bool suppressEscaping = false) { }
+        public void WriteStartArray(System.ReadOnlySpan<char> propertyName, bool suppressEscaping = false) { }
+        public void WriteStartArray(string propertyName, bool suppressEscaping = false) { }
+        public void WriteStartObject() { }
+        public void WriteStartObject(System.ReadOnlySpan<byte> propertyName, bool suppressEscaping = false) { }
+        public void WriteStartObject(System.ReadOnlySpan<char> propertyName, bool suppressEscaping = false) { }
+        public void WriteStartObject(string propertyName, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.DateTime value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.DateTimeOffset value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.Guid value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.ReadOnlySpan<byte> value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.ReadOnlySpan<char> value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, string value, bool suppressEscaping = false) { }
```

Comment:
In a re-review for JsonDocument today the argument of "if bytes the variable should say the encoding, and if not bytes it should not" was stated and received nods all around.

Reply:
> In a re-review for JsonDocument today the argument of "if bytes the variable should say the encoding, and if not bytes it should not" was stated and received nods all around.

That is because `JsonDocument` doesn't have `utf8` in the name. I believe that is the distinction here. Therefore, the `Utf8JsonWriter` doesn't need to add the `utf8` prefix to the parameter names. At least that is what we agreed on during the writer review.